app/enums/JourneyType/JourneyType.scala

```diff
@@ -18,7 +18,9 @@ package enums.JourneyType
 
 import enums.IncomeSourceJourney.IncomeSourceType
 
-case class JourneyType(operation: Operation, businessType: IncomeSourceType) {
+sealed trait JourneyType
```

Comment:
You can name is BaseJourneyType or similar,
as we have to preserver original naming convention which is